projects/backend/utils/crossword.ts

```diff
@@ -56,17 +56,22 @@ type CrosswordArticleOverrides = Pick<
 > &
     Pick<CrosswordArticle, 'crossword'>
 
-const shouldHaveSolution = (type: CrosswordType) =>
-    ![CrosswordType.EVERYMAN, CrosswordType.PRIZE].includes(type)
-
 const removeSolutionsFromEntries = (entries: Crossword['entries']) =>
     entries.map(omit(['solution']))
 
 export const patchCrossword = (
     crossword: Crossword,
     type: CrosswordType,
 ): Crossword => {
-    const solutionAvailable = shouldHaveSolution(crossword.type)
+    // for crosswords with a dateSolutionAvailable in the future, hide solutions
+    // this will be permanent even after the date has passed unless the edition is republished
+    // editions might be pressed before a crosssord is due to be published so + 1 day
+    const oneDay = 24 * 60 * 60 * 1000
+    const thisTimeTomorrow = new Date().getTime() + oneDay
```

Comment:
I'm not a fan of using the current time in the backend, because that means the output isn't as predictable anymore, it depends on some constantly-changing value. Could we instead use the expected publication time for the comparison? (Or the past publication time for re-publication.) Or, let's just never provide solutions if `dateSolutionAvailable` is non-null?